<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1257</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&amp;lt;?xml version=&amp;quot;1.0&amp;quot; encoding=&amp;quot;UTF-8&amp;quot;?&amp;gt;&lt;/p&gt;&lt;p&gt;&amp;lt;ui version=&amp;quot;4.0&amp;quot;&amp;gt;&lt;/p&gt;&lt;p&gt; &amp;lt;class&amp;gt;MainWindow&amp;lt;/class&amp;gt;&lt;/p&gt;&lt;p&gt; &amp;lt;widget class=&amp;quot;QMainWindow&amp;quot; name=&amp;quot;MainWindow&amp;quot;&amp;gt;&lt;/p&gt;&lt;p&gt;  &amp;lt;property name=&amp;quot;geometry&amp;quot;&amp;gt;&lt;/p&gt;&lt;p&gt;   &amp;lt;rect&amp;gt;&lt;/p&gt;&lt;p&gt;    &amp;lt;x&amp;gt;0&amp;lt;/x&amp;gt;&lt;/p&gt;&lt;p&gt;    &amp;lt;y&amp;gt;0&amp;lt;/y&amp;gt;&lt;/p&gt;&lt;p&gt;    &amp;lt;width&amp;gt;1257&amp;lt;/width&amp;gt;&lt;/p&gt;&lt;p&gt;    &amp;lt;height&amp;gt;732&amp;lt;/height&amp;gt;&lt;/p&gt;&lt;p&gt;   &amp;lt;/rect&amp;gt;&lt;/p&gt;&lt;p&gt;  &amp;lt;/property&amp;gt;&lt;/p&gt;&lt;p&gt;  &amp;lt;property name=&amp;quot;windowTitle&amp;quot;&amp;gt;&lt;/p&gt;&lt;p&gt;   &amp;lt;string&amp;gt;MainWindow&amp;lt;/string&amp;gt;&lt;/p&gt;&lt;p&gt;  &amp;lt;/property&amp;gt;&lt;/p&gt;&lt;p&gt;  &amp;lt;widget class=&amp;quot;QWidget&amp;quot; name=&amp;quot;centralwidget&amp;quot;&amp;gt;&lt;/p&gt;&lt;p&gt;   &amp;lt;property name=&amp;quot;styleSheet&amp;quot;&amp;gt;&lt;/p&gt;&lt;p&gt;    &amp;lt;string notr=&amp;quot;true&amp;quot;&amp;gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit,#lineEdit_2,#lineEdit_4,#lineEdit_5,#dateEdit {background-color: transparent;
background: rgba(176, 176, 176, 0.8); /* Couleur gris clair moyen */
backdrop-filter: blur(10px);
-webkit-backdrop-filter: blur(10px);
border-radius: 30px; /* Coins arrondis */
border: 2px solid rgba(255, 255, 255, 0.5);
}
#pushButton_16, #pushButton_10, #pushButton_3, #pushButton_11, 
#pushButton_5, #pushButton_4,#pushButton_17,#pushButton_18 {
    background-color: #70a2a8;
    color: white;
    border-radius: 10px;
    font-size: 12px;
    padding: 10px;
}

#pushButton_16:hover, #pushButton_10:hover, #pushButton_3:hover, #pushButton_11:hover, 
#pushButton_5:hover, #pushButton_4:hover,#pushButton_17:hover,#pushButton_18:hover
  {
    background-color: #70a2a8;  /* Change la couleur au survol */
    color: white;
}
#pushButton_16, #pushButton_10, #pushButton_3, #pushButton_11, 
#pushButton_5, #pushButton_4,#pushButton_17,#pushButton_18{
background-color: #5c878a; }</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1211</width>
      <height>841</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="styleSheet">
      <string notr="true">#pushButton_16, #pushButton_10, #pushButton_3, #pushButton_11, 
#pushButton_5, #pushButton_4,#pushButton_17,#pushButton_18 {
    background-color: #70a2a8;
    color: white;
    border-radius: 10px;
    font-size: 12px;
    padding: 10px;
}

#pushButton_16:hover, #pushButton_10:hover, #pushButton_3:hover, #pushButton_11:hover, 
#pushButton_5:hover, #pushButton_4:hover,#pushButton_17:hover,#pushButton_18:hover
  {
    background-color: #70a2a8;  /* Change la couleur au survol */
    color: white;
}
#pushButton_16, #pushButton_10, #pushButton_3, #pushButton_11, 
#pushButton_5, #pushButton_4,#pushButton_17,#pushButton_18{
background-color: #5c878a; }&lt;/string&gt;
   &lt;/property&gt;</string>
     </property>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>360</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Exportation en PDF</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>-50</y>
        <width>1281</width>
        <height>661</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/Design sans titre (2).png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_17">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>150</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>90</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/—Pngtree—delete vector icon_4070100.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>360</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>to-do-list</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>480</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>insert video</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>1131</width>
        <height>581</height>
       </rect>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="styleSheet">
       <string notr="true">#widget_3 {
    background-color: rgba(0, 0, 0, 0.4);  /* Augmenter l'opacité */
    backdrop-filter: blur(10px);
    -webkit-backdrop-filter: blur(10px);
    border-radius: 20px;
    border: 2px solid rgba(255, 255, 255, 0.7);  /* Renforcer la bordure */
}

#pushButton_16, #pushButton_10, #pushButton_3, #pushButton_11, 
#pushButton_5, #pushButton_4,#pushButton_17,#pushButton_18 {
    background-color: #70a2a8;
    color: white;
    border-radius: 10px;
    font-size: 12px;
    padding: 10px;
}

#pushButton_16:hover, #pushButton_10:hover, #pushButton_3:hover, #pushButton_11:hover, 
#pushButton_5:hover, #pushButton_4:hover,#pushButton_17:hover,#pushButton_18:hover
  {
    background-color: #70a2a8;  /* Change la couleur au survol */
    color: white;
}
#pushButton_16, #pushButton_10, #pushButton_3, #pushButton_11, 
#pushButton_5, #pushButton_4,#pushButton_17,#pushButton_18{
background-color: #5c878a; }&lt;/string&gt;
   &lt;/property&gt;</string>
      </property>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>40</y>
         <width>311</width>
         <height>481</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#widget_2{
  background-color: rgba(255, 255, 255, 0.7); /* Légèrement transparent pour bien se fondre avec l'arrière-plan */
    border-radius: 12px;
    padding: 15px;
}

#pushButton_16, #pushButton_10,#pushButton_18 {
    background-color: #70a2a8;
    color: white;
    border-radius: 10px;
    font-size: 12px;
    padding: 10px;
}

#pushButton_16:hover, #pushButton_10:hover,#pushButton_18:hover
  {
    background-color: #70a2a8;  /* Change la couleur au survol */
    color: white;
}
#pushButton_16, #pushButton_10, #pushButton_18{
background-color: #5c878a; }&lt;/string&gt;
   &lt;/property&gt;</string>
       </property>
       <widget class="QPushButton" name="pushButton_16">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>90</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer une animation</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>91</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Downloads/stat.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>230</y>
          <width>121</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Downloads/image-pdf-sigi-removebg-preview.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Statistiques</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_18">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>190</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Trier</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>130</y>
          <width>71</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Downloads/5132709.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>230</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Downloads/—Pngtree—vector checklist icon_4142453.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_28">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>360</y>
          <width>56</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Downloads/—Pngtree—vector video camera icon_4143915.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; color:#ffffff;&quot;&gt;ID:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#lineEdit{background-color: transparent;
background: rgba(176, 176, 176, 0.8); /* Couleur gris clair moyen */
backdrop-filter: blur(10px);
-webkit-backdrop-filter: blur(10px);
border-radius: 30px; /* Coins arrondis */
border: 2px solid rgba(255, 255, 255, 0.5);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#lineEdit_2 {background-color: transparent;
background: rgba(176, 176, 176, 0.8); /* Couleur gris clair moyen */
backdrop-filter: blur(10px);
-webkit-backdrop-filter: blur(10px);
border-radius: 30px; /* Coins arrondis */
border: 2px solid rgba(255, 255, 255, 0.5);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; color:#ffffff;&quot;&gt;Nom:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; color:#ffffff;&quot;&gt;Ville:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#lineEdit_4{background-color: transparent;
background: rgba(176, 176, 176, 0.8); /* Couleur gris clair moyen */
backdrop-filter: blur(10px);
-webkit-backdrop-filter: blur(10px);
border-radius: 30px; /* Coins arrondis */
border: 2px solid rgba(255, 255, 255, 0.5);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; color:#ffffff;&quot;&gt;Date :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>340</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#dateEdit {background-color: transparent;
background: rgba(176, 176, 176, 0.8); /* Couleur gris clair moyen */
backdrop-filter: blur(10px);
-webkit-backdrop-filter: blur(10px);
border-radius: 30px; /* Coins arrondis */
border: 2px solid rgba(255, 255, 255, 0.5);
}
</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>380</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; color:#ffffff;&quot;&gt;Heure:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>410</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#lineEdit_5 {background-color: transparent;
background: rgba(176, 176, 176, 0.8); /* Couleur gris clair moyen */
backdrop-filter: blur(10px);
-webkit-backdrop-filter: blur(10px);
border-radius: 30px; /* Coins arrondis */
border: 2px solid rgba(255, 255, 255, 0.5);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_15 {
    background-color: transparent;
    color: #70a2a8;
    font-size: 10pt;
    font-weight: bold;
    text-decoration: underline;
    border: none;
}</string>
       </property>
       <property name="text">
        <string>Ajouter une animation</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>460</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>show video</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>90</y>
         <width>571</width>
         <height>361</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableView {
    border: 1px solid #888;  /* Ajouter une bordure fine pour chaque cellule */
    background-color: rgba(255, 255, 255, 0.8);  /* Fond légèrement opaque */
}

QTableView::item {
    padding: 8px;  /* Ajouter de l'espace pour chaque cellule */
}

QTableView::header {
    background-color: #70a2a8;  /* Une couleur douce pour l'entête */
    color: white;  /* Couleur du texte en blanc */
}
</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>-60</x>
         <y>390</y>
         <width>281</width>
         <height>251</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../Downloads/innostay__5_-removebg-preview3.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>960</x>
         <y>40</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>905</x>
         <y>40</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../Downloads/recherche.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <zorder>widget_2</zorder>
      <zorder>label_2</zorder>
      <zorder>lineEdit</zorder>
      <zorder>lineEdit_2</zorder>
      <zorder>label_3</zorder>
      <zorder>label_15</zorder>
      <zorder>lineEdit_4</zorder>
      <zorder>label_6</zorder>
      <zorder>dateEdit</zorder>
      <zorder>label_14</zorder>
      <zorder>lineEdit_5</zorder>
      <zorder>pushButton_15</zorder>
      <zorder>tableView</zorder>
      <zorder>label_12</zorder>
      <zorder>lineEdit_3</zorder>
      <zorder>pushButton_5</zorder>
      <zorder>label_9</zorder>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>90</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/—Pngtree—vector edit profile icon_4102545.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>label</zorder>
     <zorder>widget_3</zorder>
     <zorder>pushButton_11</zorder>
     <zorder>pushButton_17</zorder>
     <zorder>label_5</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>pushButton_4</zorder>
     <zorder>label_21</zorder>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>771</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_19">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>580</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>retour</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1257</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
